src/MSBuildTaskHost/Concurrent/ConcurrentQueue.cs

```diff
@@ -554,7 +554,9 @@ internal struct Slot
     [StructLayout(LayoutKind.Explicit, Size = 192)] // padding before/between/after fields based on typical cache line size of 64
     internal struct PaddedHeadAndTail
     {
-        [FieldOffset(64)] public int Head;
-        [FieldOffset(128)] public int Tail;
+        [FieldOffset(64)]
+        public int Head;
+        [FieldOffset(128)]
```

Comment:
Nit: if they're going to be multiline let's space them too

```suggestion

        [FieldOffset(128)]
```

Reply:
Done. I also fixed some spacing near the XML comment.